### ISSUE TITLE
Save user's annotation settings

### ISSUE THREAD

**Issue by ClaudiaSchulz** [inception-project/inception#405]
The annotation settings of a user, e.g. number of lines shown, don't seem to be saved. Thus, when a user logs out and later logs in again to resume annotations, the settings are reset to the default.
Especially regarding the number of lines shown this has lead to problems as annotators forgot to change the settings every single time and therefore only annotated part of a document (only the lines shown by default).

It would be great if the settings of a user could be saved.

**Comment by reckart** [comment]
These settings should be saved per user / per project. Sounds like a bug. We'll look into it. Thanks!

**Comment by reckart** [comment]
The intended behavior is that the settings made by the user in the settings dialog of the annotation page and of the curation page are persisted. I tested it on build 5e0287797637bac2c9a641a4cc98219f62bbe42d and did not notice any problem there. I performed the following steps:

* logged in
* went to the annotation page
* changed several settings (line on screen, sidebar size, visible layers)
* confirmed the dialog, logged out
* logged back in
* went to the annotation page
* settings are all still there

(tried the same for the curation page).

Note: settings are saved **per user per project** - so if a user participates in multiple projects, settings have to be made separately for each project.

0.6.0-SNAPSHOT (2018-09-04 15:44:20, build 5e0287797637bac2c9a641a4cc98219f62bbe42d)
